### ISSUE TITLE
UWP project can't be built with arm64

### ISSUE THREAD

**Issue by yinyue200** [preemptive/protected-bugsweeper#1]
build the repo https://github.com/yinyue200/protected-bugsweeper/tree/protected-community with arm64 release
it failed and showed the error msg below

```
1>------ Build started: Project: BugSweeper, Configuration: Release Any CPU ------
1>BugSweeper -> C:\Users\username\Source\Repos\protected-bugsweeper\BugSweeper\BugSweeper\bin\Release\netstandard2.0\BugSweeper.dll
2>------ Build started: Project: BugSweeper.UWP, Configuration: Release ARM64 ------
2>  BugSweeper.UWP -> C:\Users\username\Source\Repos\protected-bugsweeper\BugSweeper\BugSweeper.UWP\bin\ARM64\Release\BugSweeper.WinUniversal.exe
2>  Running Dotfuscator with config file 'DotfuscatorConfig.xml'...
2>C:\Users\username\Source\Repos\protected-bugsweeper\PreEmptive.Dotfuscator.Xamarin\PreEmptive.Dotfuscator.Xamarin.targets(347,5): error MSB3073: The command ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\Extensions\PreEmptiveSolutions\DotfuscatorCE\dotfuscatorCLI.exe" /p:InDir="obj\ARM64\Release\DotfuscatorXamarin\dfin",OutDir="obj\ARM64\Release\DotfuscatorXamarin\dfout",ReportDir="DotfuscatorReports\ARM64\Release" "DotfuscatorConfig.xml"" exited with code 1.
========== Build: 1 succeeded, 1 failed, 1 up-to-date, 0 skipped ==========
```

**Comment by yinyue200** [comment]
Here is the details if I manually run the command
> Running Dotfuscator with config file 'Dotfuscator.xml'...
1> Dotfuscator Community Version 5.42.0.9514-e0e25f754
1> _copyright info_
1>
1> The index is outside the bounds of the array.
1>
1> Usage: dotfuscatorCLI [options] [config_file]

**Comment by Joe-Sewell-PreEmptive** [comment]
Thanks for filing this issue. We've reproduced the problem and have filed it internally.

**Comment by Joe-Sewell-PreEmptive** [comment]
This has been fixed in Dotfuscator Community 5.44.0 ([changelog](https://www.preemptive.com/support/dotfuscator-support/dotfuscator-ce-change-log/1157)) and Dotfuscator Professional 4.43.2 ([changelog](https://www.preemptive.com/support/dotfuscator-support/dotfuscator-pro-change-log/1155)). You can download the latest versions [here](https://www.preemptive.com/products/dotfuscator/downloads).
